src/openvic-simulation/dataloader/Dataloader.cpp

```diff
@@ -666,6 +666,7 @@ bool Dataloader::_load_events(GameManager& game_manager) {
 
 bool Dataloader::_load_map_dir(GameManager& game_manager) const {
 	static constexpr std::string_view map_directory = "map/";
+	static constexpr std::string_view interface_directory = "interface/"; /* TODO - use the same variable from _load_interface_files */
```

Comment:
Since nothing else in this function will use the `interface` directory and we're not going to change the region colours filepath as we need to match what Vic2 uses, might as well remove this line and just make `region_colours` hold the full path.